src/settings/lib/RuleEditor/rules-show-hint.js

```diff
@@ -309,7 +309,13 @@ class Widget {
     if (completion.options.closeOnUnfocus) {
       let closingOnBlur;
 
-      cm.on('blur', this.handleBlur = () => { closingOnBlur = setTimeout(completion.close, 100); });
+      cm.on('blur', this.handleBlur = (cmInstance, e) => {
+        const { relatedTarget } = e;
+
+        if (!relatedTarget || (relatedTarget && !isTextInputField(relatedTarget))) {
```

Comment:
Could you please add more context for this check. It can be just several new variables.